<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generatewaybillview</class>
 <widget class="QWidget" name="generatewaybillview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание накладной</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tvWaybills"/>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Склад</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leIdWarehouse"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Комплектующая</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leIdComponent"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ремонтник</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leidRepairer"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Дата накладной</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Цена детали на дату</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePriceonDate"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
